executions/graphql-kotlin-transaction-loader-instrumentation/src/main/kotlin/com/expediagroup/graphql/transactionbatcher/instrumentation/extensions/DataFetchingEnvironmentExtensions.kt

```diff
@@ -16,16 +16,19 @@
 
 package com.expediagroup.graphql.transactionbatcher.instrumentation.extensions
 
-import com.expediagroup.graphql.transactionbatcher.instrumentation.exceptions.MissingTransactionBatcherException
-import com.expediagroup.graphql.transactionbatcher.transaction.TransactionBatcher
+import com.expediagroup.graphql.transactionbatcher.instrumentation.exceptions.MissingDataLoaderRegistryException
+import graphql.GraphQLContext
 import graphql.schema.DataFetchingEnvironment
+import org.dataloader.DataLoader
+import org.dataloader.DataLoaderRegistry
 
 /**
- * get an instance of [TransactionBatcher] from the GraphQLContext
- * @return a [TransactionBatcher] instance or [MissingTransactionBatcherException] if there
- * is not a [TransactionBatcher] instance in the GraphQLContext
+ * get an instance of [DataLoaderRegistry] from the [GraphQLContext]
+ * @return [DataLoaderRegistry] instance
+ * @throws [MissingDataLoaderRegistryException] if there is not a [DataLoaderRegistry] instance in the [GraphQLContext]
  */
-fun DataFetchingEnvironment.transactionBatcher(): TransactionBatcher =
+fun <K, V> DataFetchingEnvironment.getContextDataLoader(key: String): DataLoader<K, V> =
```

Comment:
wasn't sure what `context data loader` is -> maybe this should be renamed to something like `getDataLoaderFromContext`?